src/main/java/seedu/duke/Parser.java

```diff
@@ -0,0 +1,189 @@
+package seedu.duke;
+
+import storage.Storage;
+import storage.TaskList;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.Date;
+
+public class Parser {
+
+    private final String[] cmdTypes = {"find", "list", "delete", "unmark", "mark", "todo", "event", "deadline", "bye"};
```

Comment:
```suggestion
    private final String[] CMD_TYPES = {"find", "list", "delete", "unmark", "mark", "todo", "event", "deadline", "bye"};
```
Consider changing variable name to match constant naming convention.